src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques5 {
+    private String model;
+    private List<?> messages;
```

Comment:
Consider specifying the type of List instead of using wildcard <?> for better type safety.